client/src/components/select-formula-view/SelectFormulaForm.js

```diff
@@ -114,12 +167,12 @@ const SelectFormulaForm = ({ assignments, formulas, courseId }) => {
           flexWrap: 'wrap',
           justifyContent: 'space-between'
         }}>
-          <Typography width={350} m={3}>Specify attribute values for the sub study attainnments</Typography>
+          <Typography width={350} m={3}>Specify attribute values for the sub study attainments</Typography>
           <Box sx={{ m: 3, mt: 0, alignSelf: 'flex-end',display: 'flex', lexDirection: 'column', }}>
-            <Button size='small' variant='outlined' type='submit' name='skipAttributes' sx={{ mr: 2 }}>
+            <Button size='medium' variant='outlined' type='submit' name='skipAttributes' sx={{ mr: 2 }}>
```

Comment:
Could a 'Cancel' option also be added (with `onClick => navigate(-1)`)? Because now I kind of expect that with the 'Skip for Now' button I could also cancel selecting a formula but I actually can't since I need to specify the formula and at least one sub attainment to be able to press that button.

Reply:
Cancel button added to the bottom of the form.